nntrainer/compiler/ini_interpreter.cpp

```diff
@@ -0,0 +1,322 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2021 Jihoon Lee <jhoon.it.lee@samsung.com>
+ *
+ * @file ini_interpreter.cpp
+ * @date 02 April 2021
+ * @brief NNTrainer Ini Interpreter (partly moved from model_loader.c)
+ * @see	https://github.com/nnstreamer/nntrainer
+ * @author	Jijoong Moon <jijoong.moon@samsung.com>
+ * @author	Parichay Kapoor <pk.kapoor@samsung.com>
+ * @author Jihoon Lee <jhoon.it.lee@samsung.com>
+ * @bug No known bugs except for NYI items
+ */
+#include <ini_interpreter.h>
+
+#include <vector>
+
+#include <nntrainer_error.h>
+#include <nntrainer_log.h>
+#include <parse_util.h>
+#include <util_func.h>
+
+#include <nnstreamer_layer.h>
+#include <tflite_layer.h>
+
+static constexpr const char *FUNC_TAG = "[IniInterpreter] ";
+
+static constexpr const char *UNKNOWN_STR = "Unknown";
+static constexpr const char *NONE_STR = "NONE";
```

Comment:
upper cases only here ?

Reply:
Thank you for the review, I'll update this :)

Reply:
Updated here https://github.com/nnstreamer/nntrainer/pull/1078/files . Thank you 👍